README.md

```diff
@@ -39,40 +36,34 @@ jobs:
 
 ### `enforce-sqg`
 
-If set to `true`, forces to complete execution successfully even if the failures conditions (SQG criteria) you have set are met.
+If set to `true`, the task with return a failure when security quality gates (SQG) criteria have failed.
```

Comment:
"...the task returns..."; I guess you were going for "will" here, but simple present tense says the same thing and is quicker to read/understand.